src/sentry/ratelimits/utils.py

```diff
@@ -113,13 +113,20 @@ def get_organization_id_from_token(token_id: str) -> int | None:
     return installation.organization_id if installation else None
 
 
-def get_rate_limit_config(endpoint: Type[object]) -> RateLimitConfig | None:
+def get_rate_limit_config(
+    endpoint: Type[object],
+    request: Optional[Request] = None,
+    view_args: Any = None,
+    view_kwargs: Any = None,
+) -> RateLimitConfig | None:
     """Read the rate limit config from the view function to be used for the rate limit check.
 
     If there is no rate limit defined on the endpoint, use the rate limit defined for the group
     or the default across the board
     """
     rate_limit_config = getattr(endpoint, "rate_limits", DEFAULT_RATE_LIMIT_CONFIG)
+    if callable(rate_limit_config):
```

Comment:
I don't think it's obvious to someone looking at this that their rate limits function needs to be in a certain format.
An Endpoint level function might clear that up.

Reply:
I added `request` within the kwargs as discussed!